### DISCUSSION TITLE
28.1 won't play music anymore matrix, libreelec, rpi

### DISCUSSION THREAD

**Discussion by AndrewNNZ** [Goldenfreddy0703/plugin.audio.ytmusic.exp#41] [Q&A]
Hi

28.1 won't play music from the playlist anymore. When you select a song it thinks about it for a bit and then the addon just comes up with an error.

Was working fine and nothings changed. Have done the usual clear cache, reboot, etc but hasn't helped.

Thanks

**Comment by AndrewNNZ** [comment]
Looks like this is a problem with Kodi on windows as well.

**Comment by AndrewNNZ** [comment]
I tried Beta 29 but it still comes up with an error on libreelec and windows

Thanks

**Reply by fischcode** [reply]
Please test the modified cipher.py - it's experimental but appears to help.

**Reply by AndrewNNZ** [reply]
Ok, thanks - where do I get the file from?

**Reply by Goldenfreddy0703** [reply]
Download the zip of the latest commit and extract it and put it in your kodi add-ons and all.

https://github.com/Goldenfreddy0703/plugin.audio.ytmusic.exp/archive/refs/heads/main.zip

**Comment by AndrewNNZ** [comment]
Working again! Thank you so much

**Reply by Goldenfreddy0703** [reply]
@fischcode If you could make a release on this pretty soon, that would be great.
Thanks
